src/cordova/setup.action.mjs

```diff
@@ -140,6 +141,16 @@ async function appleMacOsDebug(verbose) {
     throw new Error('Building an Apple binary requires xcodebuild and can only be done on MacOS');
   }
 
+  if (os.arch() !== 'x64') {
+    console.warn(
+      'The cordova-osx platform used to build the MacOS client is deprecated, and thus has not added support for your CPU architecture. Your results may vary.'
+    );
+  }
+
```

Comment:
As discussed, this is not quite accurate. The macOS client does run on arm64, though we have issues with the debug web build for some reason.

```suggestion
```